### ISSUE TITLE
`apt-get update` failing in `pkg` job on Debian Stretch 

### ISSUE THREAD

**Issue by ximon18** [NLnetLabs/ploutos#72]
From: https://github.com/NLnetLabs/ploutos-testing/actions/runs/4775398141/jobs/8510418085#step:8:86

_(a re-run of a job that succeeded two days ago)_

```
+ apt-get update
Ign:1 http://deb.debian.org/debian stretch InRelease
Ign:2 http://deb.debian.org/debian stretch-updates InRelease
Ign:3 http://deb.debian.org/debian stretch Release
Ign:4 http://security.debian.org/debian-security stretch/updates InRelease
Ign:5 http://deb.debian.org/debian stretch-updates Release
Ign:6 http://security.debian.org/debian-security stretch/updates Release
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Ign:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Ign:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Ign:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Err:8 http://security.debian.org/debian-security stretch/updates/main amd64 Packages
  404  Not Found [IP: 151.101.2.132 80]
Ign:10 http://security.debian.org/debian-security stretch/updates/main all Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Ign:9 http://deb.debian.org/debian stretch/main amd64 Packages
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
Ign:7 http://deb.debian.org/debian stretch/main all Packages
Err:9 http://deb.debian.org/debian stretch/main amd64 Packages
  404  Not Found
Ign:11 http://deb.debian.org/debian stretch-updates/main all Packages
Err:12 http://deb.debian.org/debian stretch-updates/main amd64 Packages
  404  Not Found
Reading package lists...
W: The repository 'http://deb.debian.org/debian stretch Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-updates Release' does not have a Release file.
W: The repository 'http://security.debian.org/debian-security stretch/updates Release' does not have a Release file.
E: Failed to fetch http://deb.debian.org/debian/dists/stretch/main/binary-amd64/Packages  404  Not Found
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-updates/main/binary-amd64/Packages  404  Not Found
E: Failed to fetch http://security.debian.org/debian-security/dists/stretch/updates/main/binary-amd64/Packages  404  Not Found [IP: 151.101.2.132 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.
Error: Process completed with exit code 100.
```

**Comment by ximon18** [comment]
Cause: https://lists.debian.org/debian-devel-announce/2023/03/msg00006.html

**Comment by ximon18** [comment]
Fixed in [v7.0.2](https://github.com/NLnetLabs/ploutos/releases/tag/v7.0.2) and also back ported to [v6.3.1](https://github.com/NLnetLabs/ploutos/releases/tag/v6.3.1).
